scripts/pre_commit_linter.py

```diff
@@ -343,9 +358,51 @@ def _pre_commit_linter():
     print '\n'.join(summary_messages)
     print ''
 
+    linting_status = 0
     if any([message.startswith(_MESSAGE_TYPE_FAILED) for message in
```

Comment:
You might as well reuse this logic for the bad patterns -- start the error message with _MESSAGE_TYPE_FAILED, and maybe move this final check to main(). What do you think? Then you don't have to pass linting_status all over the place.